ci-operator/step-registry/openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-chain.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "openshift-qe/kube-burner-smoke/openshift-qe-kube-burner-smoke-ref.yaml",
+	"path": "openshift-qe/control-plane-smoke/openshift-qe-control-plane-smoke-chain.yaml",
```

Comment:
Should this be control-plane-smoke-chain or control-plane-smoke-ref?

Reply:
it should be chain, this file gets automatically created so if there was an issue with it it would show in the checks

Reply:
Hmm.. true.. not sure why metadata got changed from ref to chain, if all we are doing is rename `kube-burner` to `control-plane`

Reply:
Looks like the old ref file thought the name changed to the chain and vise versa. If you look at the end of the name of the file. it should match the path

Reply:
> Looks like the old ref file thought the name changed to the chain and vise versa. If you look at the end of the name of the file. it should match the path

Yes, the name of the file matches the pattern, checks are clean.
lgtm, unless you say otherwise!